src/main/c/zlib/extconf.rb

```diff
@@ -6,7 +6,57 @@
 #
 
 require 'mkmf'
+require 'rbconfig'
```

Comment:
We're taking a fair bit of code from MRI here. You need to add the license header & attribution above this.

Reply:
@chrisseaton? We do not have it in other `extconf.rb`.

Reply:
Other `extconf.rb` under `src/main/c` are our own, except `syslog/extconf.rb`.

Reply:
Might as well put the MRI licence header in this.

Reply:
ok

Reply:
fixed